src/Microsoft.ML.Transforms/ProduceIdTransform.cs

```diff
@@ -60,7 +60,7 @@ public static Bindings Create(ModelLoadContext ctx, Schema input)
                 return new Bindings(input, true, name);
             }
 
-            public void Save(ModelSaveContext ctx)
+            internal void Save(ModelSaveContext ctx)
```

Comment:
>internal [](start = 12, length = 8)

Consternation, uproar!

Reply:
Resignation!

---
In reply to: [254489466](https://github.com/dotnet/machinelearning/pull/2431#discussion_r254489466) [](ancestors = 254489466)